macros/edr/system/system_utils/empty_table.sql

```diff
@@ -26,7 +26,8 @@
     ('failures', 'bigint'),
     ('test_short_name', 'string'),
     ('test_alias', 'string'),
-    ('result_rows', 'long_string')
+    ('result_rows', 'long_string'),
+    ('failed_count', 'bigint')
```

Comment:
I think it should be `failed_row_count` or `row_count_failures`, otherwise it's really confusing compared to `failures`.

Reply:
on it